drivers/gpio/gpio_stm32.c

```diff
@@ -717,6 +735,10 @@ static int gpio_stm32_init(const struct device *dev)
 	/* Cf: L4/L5 RM, Chapter "Independent I/O supply rail" */
 	LL_PWR_EnableVddIO2();
 	z_stm32_hsem_unlock(CFG_HW_RCC_SEMID);
+#elif defined(CONFIG_SOC_SERIES_STM32H7RSX)
+	/* TBC : stm32Cube does */
+	LL_PWR_EnableXSPIM1();
+	LL_PWR_EnableUSBVoltageDetector();
```

Comment:
Does it really relate to exti ?
Should it be in soc ? xspi ?

Reply:
useless, removing